<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="viewerPosition">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Viewer Position</string>
    </property>
    <widget class="QLabel" name="viewer_xlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="viewer_ylabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="viewer_zlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="viewer_xtext">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="viewer_ytext">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="viewer_ztext">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="lightPosition">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Light Position</string>
    </property>
    <widget class="QLabel" name="light_xlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="light_ylabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="light_zlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="light_xtext">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="light_ytext">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="light_ztext">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="positionalLightEnable">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Positional</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="directionalLightEnable">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Directional</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="lights">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>1181</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Lights </string>
    </property>
    <widget class="QGroupBox" name="diffused">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>271</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Diffused</string>
     </property>
     <widget class="QSlider" name="diffused_Gslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="diffused_Rslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="diffused_Rlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QSlider" name="diffused_Bslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="diffused_Glabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="diffused_Alabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="diffused_Blabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QSlider" name="diffused_Aslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="diffused_enable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="specular">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>271</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Specular</string>
     </property>
     <widget class="QSlider" name="specular_Gslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="specular_Rslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="specular_Rlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QSlider" name="specular_Bslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="specular_Glabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="specular_Alabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="specular_Blabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QSlider" name="specular_Aslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="specular_enable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="spotlight">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>241</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>SpotLight</string>
     </property>
     <widget class="QRadioButton" name="spotlight_enable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="spot_xlabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spot_ztext">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-1.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spot_xtext">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>-1.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="spot_ylabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spot_ytext">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-1.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="spot_zlabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QSlider" name="spot_cutoffSlider">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="spot_cutoffLabel">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>130</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CutOff</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spot_exponentText">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>2.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="spot_exponentLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Exponent</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ambient">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>30</y>
       <width>271</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Ambient</string>
     </property>
     <widget class="QSlider" name="ambient_Gslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="ambient_Rslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="ambient_Rlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QSlider" name="ambient_Bslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="ambient_Glabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="ambient_Alabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="ambient_Blabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QSlider" name="ambient_Aslider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="ambient_enable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="ok">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>176</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="GLWidget" name="openglWidget" native="true">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>479</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>521</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>BrowseFile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputFileText"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
